src/handler.js

```diff
@@ -114,11 +116,45 @@ const handle = async message => {
       Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
     },
   }).then(response => {
-    response.data.pipe(resizer).pipe(writeStream);
+    response.data.pipe(resizer).pipe(resizedStream);
+  });
+
+  let b64Chunks = [];
+
+  resizedStream.pipe(b64Stream);
+
+  b64Stream.pipe(new Base64Encode());
+
+  b64Stream.on('data', chunk => {
+    b64Chunks.push(chunk);
+  });
+  const [owner, repo] = process.env.GITHUB_REPO.split('/');
+  const branch = process.env.GITHUB_REPO_BRANCH;
+  const emojiRepoDir = process.env.GITHUB_REPO_DIR;
+  const emojiPath = path.join(emojiRepoDir, `${emojiAlias}.${filetype}`);
+
+  b64Stream.on('finish', async () => {
+    console.log('b64', Buffer.concat(b64Chunks).toString('base64'));
```

Comment:
Leftover?

If you're intentionally leaving it here then 1. use poor winston which you painstakingly implemented, 2. don't concat and toString twice as it's an expensive operation for gifs/big files.